R/utils.R

```diff
@@ -77,3 +77,19 @@ replace_na <- function(data, replace = NA, ...) {
   }
   data
 }
+
+get_os <- function() {
```

Comment:
I see only one usage of this, and all it does is checking for `osx`. Why not simplify it to `is_osx()`?

Reply:
done